[2022-09-06T15:00:30.018+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2022-07-05T00:00:00+00:00 [queued]>
[2022-09-06T15:00:30.034+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2022-07-05T00:00:00+00:00 [queued]>
[2022-09-06T15:00:30.034+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-06T15:00:30.034+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 6
[2022-09-06T15:00:30.034+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-06T15:00:30.055+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2022-07-05 00:00:00+00:00
[2022-09-06T15:00:30.061+0000] {standard_task_runner.py:52} INFO - Started process 609 to run task
[2022-09-06T15:00:30.066+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks_v01', 'postgres_to_s3', 'scheduled__2022-07-05T00:00:00+00:00', '--job-id', '512', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_hooks.py', '--cfg-path', '/tmp/tmpyfy_jatl', '--error-file', '/tmp/tmp6poyzrcf']
[2022-09-06T15:00:30.066+0000] {standard_task_runner.py:80} INFO - Job 512: Subtask postgres_to_s3
[2022-09-06T15:00:30.189+0000] {task_command.py:371} INFO - Running <TaskInstance: dag_with_postgres_hooks_v01.postgres_to_s3 scheduled__2022-07-05T00:00:00+00:00 [running]> on host 45495cb94d52
[2022-09-06T15:00:30.299+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=James
AIRFLOW_CTX_DAG_ID=dag_with_postgres_hooks_v01
AIRFLOW_CTX_TASK_ID=postgres_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T00:00:00+00:00
[2022-09-06T15:00:30.312+0000] {base.py:68} INFO - Using connection ID 'postgres_localhost' for task execution.
[2022-09-06T15:00:30.321+0000] {dag_with_postgres_hooks.py:29} INFO - save orders data in text file get_orders.txt
[2022-09-06T15:00:30.322+0000] {python.py:173} INFO - Done. Returned value was: None
[2022-09-06T15:00:30.340+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks_v01, task_id=postgres_to_s3, execution_date=20220705T000000, start_date=20220906T150030, end_date=20220906T150030
[2022-09-06T15:00:30.365+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-06T15:00:30.414+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
